hack/cosign-verify-image.sh

```diff
@@ -40,12 +40,27 @@ logical_image="${1}"
 physical_image="${2}"
 
 echo -ne "Validating ${logical_image} ... "
-for key_url in "${HPE_OCI_SIGNING_KEYS[@]}"; do
-    key=$(basename "${key_url}")
-    if cosign verify --key "${BUILDDIR}/security/keys/oci/${key}" --insecure-ignore-tlog --insecure-ignore-sct "${physical_image}"  2>/dev/null 1>/dev/null; then
+tmpdir=$(mktemp -d)
+trap "rm -rf ${tmpdir}" EXIT
+
+# cosign is not capable of reading multiple keys from signle file, will need to toss
+# keys into separate files
+count=0
+while read -r line; do
+    if [ "${line}" == '-----BEGIN PUBLIC KEY-----' ]; then
```

Comment:
When using `[`, string comparison is just `=`, not `==`.  Some implementations accept `==` instead of `=`, but that is not standard.  However if you switch to using the bash built-in `[[` instead of `[`, then `==` is correct for string comparison.

Reply:
Ah, but what that document does say is:
```
‘=’ should be used with the test command for POSIX conformance.
```
That's what I was remembering.